### PR TITLE
Upgrade rclone to v1.61.1

### PR DESCRIPTION
**Pull request by tesshuflower**
**Describe what this PR does**

**Is there anything that requires special attention?**

**Related issues:**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - VolSync now uses a single container image for the controller and all movers
+- Rclone upgraded to v1.60.1
```
Comment (JohnStrunk) [inline CHANGELOG.md:19]:
```suggestion
- Rclone upgraded to v1.61.1
```

Comment (tesshuflower) [reply CHANGELOG.md]: yikes! Doing too many things at once - thx John

**Review by JohnStrunk — COMMENTED** [review]
/lgtm

**Comment by tesshuflower** [comment]
/cc @JohnStrunk 

Sorry, do you mind reviewing again? Had to make another commit to resolve a merge conflict in CHANGELOG

**Review by JohnStrunk — COMMENTED** [review]
/lgtm

**Pull request merged**